### PR TITLE
Add Simple smart answer schema

### PR DESCRIPTION
**Pull request by davidbasalla**
Also added definitions for nodes and node options.

Generate simple smart answer schema.

Also regenerate all other schemas as we added definitions.

https://trello.com/c/2ucbq949/643-migrate-simple-smart-answer-format-to-rendered

### CODE REVIEW

File: formats/definitions.json [hunk 1]
```diff
@@ -142,6 +142,51 @@
...
+      "type": "array",
+      "items": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "title"
+        ],
+        "properties": {
+          "kind": {
+            "type": "string"
```
Comment (cbaines) [inline formats/definitions.json]: If kind has a few valid values, an enum [1] might be better.

1: http://json-schema.org/latest/json-schema-validation.html#rfc.section.5.20

**Comment by davidbasalla** [comment]
thanks @cbaines, have updated the definition!

File: formats/definitions.json [hunk 2]
```diff
@@ -142,6 +142,54 @@
         }
       }
     },
+    "nodes": {
```
Comment (fofr) [inline formats/definitions.json]: Does this need to be a definition if it's only used in simple smart answers?

This data structure is complex and needs some description fields to explain it.

**Comment by davidbasalla** [comment]
Thanks for the review @fofr, have removed the definition as it would only be used by simple smart answers and have added descriptions

**Comment by fofr** [comment]
Could you include some real world examples?

**Comment by davidbasalla** [comment]
@fofr have replaced the example with a real world example! Is the general idea to always add a real example - rather than a toy one?)

**Comment by fofr** [comment]
@davidbasalla Yeah, it's best to have examples that cover the range of real world uses.

File: formats/simple_smart_answer/frontend/examples/simple-smart-answer.json [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+  "description": "Find out if you can become an approved driving instructor (ADI) and what you need to do to qualify",
+  "details": {
+    "start_button_text": "Start now",
+    "body": "<p>Find out if you can become an approved driving instructor (<abbr title=\"approved driving instructor\">ADI</abbr>) in Great Britain and what you need to do to qualify.</p>\n\n<div role=\"note\" aria-label=\"Information\" class=\"application-notice info-notice\">\n<p>There’s a different process in <a rel=\"external\" href=\"http://www.nidirect.gov.uk/index/information-and-services/motoring/driving-for-a-living/driving-instructors/how-to-become-an-approved-driving-instructor-adi.htm\">Northern Ireland.</a></p>\n</div>\n\n",
+    "nodes": [
+      {
+        "kind": "question",
+        "slug": "question-1",
+        "title": "Are you already registered as a driving instructor in another EU country?",
+        "body": "\n",
```
Comment (fofr) [inline formats/simple_smart_answer/frontend/examples/simple-smart-answer.json]: This feels wrong to me

**Comment by davidbasalla** [comment]
Good catch @fofr, have updated the presenter to send the `body` field for nodes only when present.

**Pull request merged**